vtex/loaders/intelligentSearch/productListingPage.ts

```diff
@@ -250,18 +250,41 @@ const loader = async (
   const url = new URL(baseUrl);
   const segment = ctx.bag.get(SEGMENT);
   const currentPageoffset = props.pageOffset ?? 1;
-  const { selectedFacets: baseSelectedFacets, page, ...args } = searchArgsOf(
-    props,
-    url,
-  );
+  const {
+    selectedFacets: baseSelectedFacets,
+    page,
+    ...args
+  } = searchArgsOf(props, url);
   const pageTypesPromise = pageTypesFromPathname(url.pathname, ctx);
   const selectedFacets = baseSelectedFacets.length === 0
     ? filtersFromPathname(await pageTypesPromise)
     : baseSelectedFacets;
 
   const selected = withDefaultFacets(selectedFacets, ctx);
   const fselected = selected.filter((f) => f.key !== "price");
+  const pageTypes = await pageTypesPromise;
+
+  const pathList = url.pathname.split("/").slice(1);
+
+  /**
+   *  verify if when url its not a category/department/collection
+   *  and is not a default query format but is a valid search based in default IS behavior on native stores
+   */
+  if (
+    !pageTypes.length &&
+    !selected.length &&
+    !args.query &&
+    pathList.length == 1
+  ) {
```

Comment:
Create a variable with this condition and name it appropriately

Reply:
done.